hr_course/models/hr_course_attendee.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HRCourseAttendee(models.Model):
+    _name = "hr.course.attendee"
+    _description = "Course Attendee"
+
+    course_schedule_id = fields.Many2one(
+        "hr.course.schedule", ondelete="cascade", readonly=True, required=True
+    )
+    course_id = fields.Many2one(related="course_schedule_id.course_id")
+    course_validity_end_date = fields.Date(
+        related="course_schedule_id.validity_end_date"
+    )
+
+    name = fields.Char(related="course_schedule_id.name", readonly=True)
+    alerted = fields.Boolean(help="Shows if notification email for course was sent")
```

Comment:
1. This is a new field, but it is not used anywhere apparently
2. If I missed the usefulness of the field, please rename it to explicit what kind of notification this represents : course expiration notification, course inscription, course start?

Reply:
@gurneyalex  it is used in cron to distinguish courses for which alert already sent.